docs/src/main/sphinx/connector/hive.rst

```diff
@@ -426,6 +426,10 @@ with ORC files performed by the Hive connector.
         accessed by their ordinal position in the Hive table definition. The
         equivalent catalog session property is ``orc_use_column_names``.
       - ``false``
+    * - ``hive.orc.bloom-filters.enabled``
+      - Enable ``orc_bloom_filter_*`` :ref:`table
```

Comment:
Just "Enable bloom filters for predicate pushdown" looks fine to me.

Also, I don't think using wildcard like above is good idea. Such sentence is weak for future changes. For instance, if we add new `orc_bloom_filter_` configuration property in the future, the sentence might get stale.